buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf

```diff
@@ -4,44 +4,20 @@
   "play_wav_cmdline": "paplay %1",
   "play_mp3_cmdline": "mpg123 %1",
   "ipc_path": "/dev/shm/mycroft/ipc/",
-  "enclosure": {
-    "platform": "OpenVoiceOS",
-    "ntp_sync_on_boot": true
```

Comment:
Does it take care of ntp sync by default?

Reply:
oops, no it doesnt

Reply:
Hmm, strange that I then didn't notice it. I am not starting the enclosure service anymore, so either way this should have not been used.

Let me double check.

Reply:
its used by skills service DevicePrimer

Reply:
So, am I correct then to better leave this for now?